lib/superset/dashboard/datasets/list.rb

```diff
@@ -7,14 +7,15 @@ module Superset
   module Dashboard
     module Datasets
       class List < Superset::Request
-        attr_reader :id  # dashboard id
+        attr_reader :id, :include_filter_datasets # dashboard id
 
         def self.call(id)
           self.new(id).list
         end
 
-        def initialize(id)
+        def initialize(id, include_filter_datasets = false)
```

Comment:
@soundarya-mv 
as we are now passing more than one param, its probably a good idea to make these named attributes
```
  def initialize(dashboard_id: dashboard_id, include_filter_datasets: false)
```